.yarn/sdks/typescript/lib/tsserver.js

```diff
@@ -109,15 +109,13 @@ const moduleWrapper = tsserver => {
         return process.platform === `win32`
           ? str.replace(/^.*zipfile:\//, ``)
           : str.replace(/^.*zipfile:/, ``);
-      } break;
-
+      }
       case `vscode`:
       default: {
         return process.platform === `win32`
           ? str.replace(/^\^?(zip:|\/zip)\/+/, ``)
           : str.replace(/^\^?(zip:|\/zip)\/+/, `/`);
-      } break;
-    }
+      }    }
```

Comment:
ditto for this break after return.

Two braces on a line is a bit weird, but some style thing did it, so... 🤷‍♂️